### ISSUE TITLE
Crash with ruby 2.2.2 for nanoc 4.0.1

### ISSUE THREAD

**Issue by nrdufour** [nanoc/nanoc#741]
Hello,

I'm using nanoc in a ruby 2.2.2 docker container and I get this error for anything:
```
Captain! We’ve been hit!

Message:

SyntaxError: /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli/commands/create-site.rb:21: invalid multibyte char (US-ASCII)
/usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli/commands/create-site.rb:18: syntax error, unexpected end-of-input, expecting tSTRING_CONTENT or tSTRING_DBEG or tSTRING_DVAR or tSTRING_END
    DEFAULT_CONFIG = <<EOS unless defined? DEFAULT_CONFIG
                          ^

Compilation stack:

  (empty)

Stack trace:

  0. /usr/local/bundle/gems/cri-2.7.0/lib/cri/command.rb:98:in `instance_eval'
  1. /usr/local/bundle/gems/cri-2.7.0/lib/cri/command.rb:98:in `define'
  2. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli.rb:156:in `load_command_at'
  3. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli.rb:110:in `block in setup_commands'
  4. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli.rb:108:in `each'
  5. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli.rb:108:in `setup_commands'
  6. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli.rb:86:in `setup'
  7. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli.rb:52:in `block in run'
  8. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli/error_handler.rb:64:in `handle_while'
  9. /usr/local/bundle/gems/nanoc-4.0.1/lib/nanoc/cli/error_handler.rb:22:in `handle_while'
  ... 4 more lines omitted. See full crash log for details.

If you believe this is a bug in Nanoc, please do report it at
-> https://github.com/nanoc/nanoc/issues/new <-

A detailed crash log has been written to ./crash.log.
```

Suggestions?

Nicolas

**Comment by ddfreyne** [comment]
I managed to reproduce this by forcing the default external encoding to ASCII and attempting to load a command. Fix coming up in 4.0.2.

**Comment by ddfreyne** [comment]
A workaround is setting your environment’s encoding to UTF-8 before invoking nanoc, e.g.

```
export LC_ALL=C.UTF-8
```

**Comment by ddfreyne** [comment]
Fixed in #742.

**Comment by nrdufour** [comment]
Thank you! The workaround is working perfectly!
